conf/application.conf

```diff
@@ -197,6 +197,13 @@ ot {
   }
 }
 
+ot.meta.apiVersion.x=${?META_APIVERSION_MAJOR}
```

Comment:
Will this use the earlier value as default value in case the corresponding environment variable is not defined?

Reply:
That's correct